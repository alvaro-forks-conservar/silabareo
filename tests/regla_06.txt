subsanar
taxi
xenófobo
cansado
pulso
persona
adsorción
marxista