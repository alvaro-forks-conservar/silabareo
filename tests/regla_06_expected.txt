sub-sa-nar
ta-xi
xe-nó-fo-bo
can-sa-do
pul-so
per-so-na
ad-sor-ción
mar-xis-ta